### PR TITLE
[Site] refactor commonmark config

### PR DESCRIPTION
**Pull request by kbond**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Issues        | n/a
| License       | MIT

This change allows us to add commonmark configuration to the extensions. Also, I refactored the Changelog to use [native commonmark extension](https://commonmark.thephpleague.com/2.4/extensions/mentions/) for github mentions/issue links (fixes a few bugs too).

### CODE REVIEW

**Comment by smnandre** [comment]
> Also, I refactored the Changelog to use [native commonmark extension](https://commonmark.thephpleague.com/2.4/extensions/mentions/) for github mentions/

Wow ! 

> fixes a few bugs too

❤️

File: ux.symfony.com/src/Service/CommonMark/ConverterFactory.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    'prefix' => '#',
+                    'pattern' => '\d+',
+                    'generator' => 'https://github.com/symfony/ux/issues/%d',
+                ],
+            ],
+        ]);
+
+        $converter->getEnvironment()
+            ->addRenderer(FencedCode::class, new CodeRenderer(), 10)
+            ->addExtension(new ExternalLinkExtension())
```
Comment (smnandre) [inline ux.symfony.com/src/Service/CommonMark/ConverterFactory.php:45]: Can we restrict a bit the URLs ?

**Review by smnandre — APPROVED** [review]

File: ux.symfony.com/src/Service/CommonMark/ConverterFactory.php (hunk 1 above)
Comment (kbond) [reply ux.symfony.com/src/Service/CommonMark/ConverterFactory.php:45]: [I believe so yes](https://commonmark.thephpleague.com/2.4/extensions/external-links/#configuration), what did you have in mind?

Comment (smnandre) [reply ux.symfony.com/src/Service/CommonMark/ConverterFactory.php:45]: No you're right let's wait _if_ this is needed :)

**Review by smnandre — APPROVED** [review]

File: ux.symfony.com/src/Service/CommonMark/ConverterFactory.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                'github_issue' => [
+                    'prefix' => '#',
+                    'pattern' => '\d+',
+                    'generator' => 'https://github.com/symfony/ux/issues/%d',
+                ],
+            ],
+        ]);
+
+        $converter->getEnvironment()
+            ->addRenderer(FencedCode::class, new CodeRenderer(), 10)
```
Comment (Kocal) [inline ux.symfony.com/src/Service/CommonMark/ConverterFactory.php:44]: Soon Tempest's highlighting :eyes:

**Pull request merged**